src/Rings.jl

```diff
@@ -14,9 +14,19 @@ end
 #
 ###############################################################################
 
+@doc Markdown.doc"""
```

Comment:
I believe we are now not introducing docstrings for each instance of a function across the library, but simply having one main dosctring which says what e.g. elem_type or parent_type does so that when searching for it at the REPL one doesn't get dozens of instances of the same docs.

So, just wanted to check if this is the main instance of elem_type docs in the whole library. If not I would say it can be removed. (It's admittedly a little hard to assign a canonical place for documentation for a function like this, as it only applies to concrete types and can't be put in Rings.jl, say.)

Reply:
Perhaps we should come up with a way to annotate this as the canonical instance of these docs.

Reply:
I do realise the docs were already there, so I'm merely suggesting that we take the opportunity to sort that issue out at the same time, if it doesn't delay anything.

Reply:
If i understand you correctly that's exactly what this PR is doing:
before:
```julia
help?> parent_type
search: parent_type

  parent_type(::Type{<:Perm})

  Return the type of the parent of a permutation.

```
after:
```julia
help?> parent_type
search: parent_type

  parent_type(element)
  parent_type(element_type)

  Given an element (or its type), return the type of its parent object.
```

Reply:
Yes, I agree it is.

I just checked and your docs for parent_type and elem_type are the only ones in AbstractAlgebra for those functions anyway.